src/resources/Fields/Fields.ts

```diff
@@ -42,4 +42,8 @@ export default class Field extends Resource {
     list(params?: ListFieldsParams) {
         return this.api.get<PageModel<FieldModel>>(this.buildPath(`${Indexes.baseUrl}/page/fields`, params));
     }
+
+    search(params?: FieldListingOptions) {
+        return this.api.get<PageModel<FieldModel>>(this.buildPath(`${Indexes.baseUrl}/search`, params));
```

Comment:
I think it should be

```suggestion
        return this.api.get<PageModel<FieldModel>>(this.buildPath(`${Field.baseUrl}/search`, params));
```